frodo.js

```diff
@@ -5,124 +5,122 @@
  * for the extras/ plugins if they are loaded into a presentation.
```

Comment:
SCRIPT-I'm impressed with your use of design patterns here, it really enhances the code organization.